pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go

```diff
@@ -57,8 +57,8 @@ func balancedResourceScorer(requested, allocable *schedulercache.Resource, inclu
 		return int64((1 - variance) * float64(schedulerapi.MaxPriority))
 	}
 
-	if cpuFraction >= 1 || memoryFraction >= 1 {
-		// if requested >= capacity, the corresponding host should never be preferred.
+	if cpuFraction > 1 || memoryFraction > 1 {
```

Comment:
If pod's cpu and  mem requested equals to allocatable, this means this node is suitable for the pod in `balancedResource` policy, we deal with this case just like `mostResource` and `leastResource` policies. WDYT?

Reply:
The main reason of moving equal sign out is that we should give priority for the node whose cpu or memory is equal to pod's request, in order to prevent the case all nodes' balancedResource score are 0 to happen.

Reply:
hm... give me sometime to check this.

If anyone knows the detail , please help to comments :)